### PR TITLE
[NO-TASK]: webhook param on Payment endpoint

### PR DESCRIPTION
**Pull request by Danswar**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

**Comment by TaprootFreak** [comment]
👀

**Comment by Yannick1712** [comment]
Waiting for @davidleomay

File: src/subdomains/core/payment-link/dto/create-payment-link-payment.dto.ts [hunk 1]
```diff
@@ -29,4 +29,9 @@ export class CreatePaymentLinkPaymentDto {
   @IsDate()
   @Type(() => Date)
   expiryDate?: Date;
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsString()
+  webhookUrl?: string;
```
Comment (davidleomay) [inline src/subdomains/core/payment-link/dto/create-payment-link-payment.dto.ts:36]: - [ ] Add webhookUrl to payment entity
- [ ] Use URL from payment, if it exists

Comment (Danswar) [reply src/subdomains/core/payment-link/dto/create-payment-link-payment.dto.ts]: My mistake, I understand the suggestion, but my intention was to use the webhook in the PaymentLink, not in the PaymentLinkPayment.

I updated the PR to reflect that so I think these suggestions no longer apply, thanks for the observation.

**Comment by Danswar** [comment]
@davidleomay ready for a second look

**Review by davidleomay — APPROVED** [review]

File: src/subdomains/core/payment-link/dto/create-payment-link-payment.dto.ts (hunk 1 above)
Comment (davidleomay) [reply src/subdomains/core/payment-link/dto/create-payment-link-payment.dto.ts]: Yes, good idea 👍

**Comment by Danswar** [comment]
@davidleomay solved merge conflicts

**Comment by TaprootFreak** [comment]
@Danswar please use "Squash and merge"

not, "Merge pull request"

**Comment by davidleomay** [comment]
> Where are my Merge buttons? 😅😂
> 
> Screenshot 2024-09-30 at 12 38 37 PM

Haha 🤣

**Pull request merged**